### DISCUSSION TITLE
Find Window in System Informer - How to use?

### DISCUSSION THREAD

**Discussion by Almighty188** [winsiderss/systeminformer#2182] [Questions / Answers] [answered]
I have a question, how does one use the Find Window and what is supposed to happen?  I know the Find Window and thread works as it will show the process but what does Find Window do exactly?

**Comment by dmex** [comment, marked as answer]
It selects the process that owns the selected window. You click and drag the button to find the window and then drop to select the window.

**Reply by Almighty188** [reply]
Thanks @dmex!
